src/execution/expression_executor.cpp

```diff
@@ -26,7 +26,7 @@ ExpressionExecutor::ExpressionExecutor(vector<unique_ptr<Expression>> &exprs) :
 
 void ExpressionExecutor::AddExpression(Expression &expr) {
 	expressions.push_back(&expr);
-	auto state = make_unique<ExpressionExecutorState>();
+	auto state = make_unique<ExpressionExecutorState>(expr.ToString());
```

Comment:
It could be moved to ```Initialize``` and being cleaner. But the ```Initialize``` function initializes the ExpressionExecutor object, not the ExpressionExecutorState. It could become buggy since, after any ExpressionExecutorState initiation, there should be an ExpressionExecutor and get initialized. Off-course, the ExpressionExecutorState is only used here.